web/src/content/posts.ts

```diff
@@ -0,0 +1,143 @@
+import * as authors from './authors'
+import type { Author } from './authors'
+
+export type Post = {
+  slug: string
+  title: string
+  brief: string
+  tags: string[]
+  publishedAt: Date
+  author: Author
+  imageUrl?: string
+}
+
+const posts: Post[] = [
+  {
+    slug: 'bighorn-update',
+    title: 'Bighorn Update',
+    brief:
+      "Nine months ago, the RedwoodJS team decided to go all-in on React Server Components. The first version of Redwood that fully supports RSC will begin the Bighorn epoch. Today I'm excited to bring you an update on our work towards that goal.",
+    tags: ['redwoodjs'],
```

Comment:
I'll follow up with another PR to implement using these tags. We can talk about everywhere we'd want them to appear.